cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -131,6 +133,35 @@ def test_embedded_ansible_repository_crud(ansible_repository, wait_for_ansible):
     assert view.description.value == updated_description
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_repository_branch_crud(appliance, wait_for_ansible):
+    """
+    Ability to add repo with branch (without SCM credentials).
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/12h
+        tags: ansible_embed
+    """
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
+        name=fauxfactory.gen_alpha(),
+        url="https://github.com/ManageIQ/integration_tests_playbooks.git",
```

Comment:
I think better to pick this URL from yamls not hardcore here... `ansible_links.embedded_ansible`

Reply:
Yup. I will update it.